spotify_player/src/command.rs

```diff
@@ -294,8 +296,7 @@ impl Command {
             Self::Repeat => "cycle the repeat mode",
             Self::ToggleFakeTrackRepeatMode => "toggle fake track repeat mode",
             Self::Shuffle => "toggle the shuffle mode",
-            Self::VolumeUp => "increase playback volume by 5%",
-            Self::VolumeDown => "decrease playback volume by 5%",
+            Self::VolumeChange { offset: _ } => "change the value by an offset",
```

Comment:
should include the offset in the string documentation

Reply:
Do you mean as in include the default?

Reply:
```suggestion
            Self::VolumeChange { offset } => format!("change the value by {offset} percent"),
```

you might also need to change the function's signature to be `pub fn desc(self) -> String`

Reply:
If I change the signature to `pub fn desc(self) -> String` then I will have to call `.to_string()` on all of the `str`s. Can you think of a better way of doing this?

Reply:
you can handle the commands with arguments separately, then for other commands, you can do

```rust
match {
   ...
}.to_string()
```